### DISCUSSION TITLE
Remote Control slave machine when using gridder

### DISCUSSION THREAD

**Discussion by bsjdsp** [apohl79/audiogridder#561] [General]
Let’s say I wanted to use my master machine to control my workstation that hosts the plugins on. Is there a good safe offline program (home wire networks) to control my other computer/stream the screen on my master machine in case I need to do anything outside of gridder that way I dont need to keep unplugging my HDMI or display cables to go back and forth.

**Comment by CaBuffalo** [comment]
VNC can do the job for you.

On Mon, May 17, 2021 at 4:12 PM bsjdsp ***@***.***> wrote:

> Let’s say I wanted to use my master machine to control my workstation that
> hosts the plugins on. Is there a good safe offline program (home wire
> networks) to control my other computer/stream the screen on my master
> machine in case I need to do anything outside of gridder that way I dont
> need to keep unplugging my HDMI or display cables to go back and forth.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/561>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BLEDJOSVA6KZD5QCDTOGPGLANCNFSM45BL6HDA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by bsjdsp** [reply]
Does Vnc work offline, I have tried Google remote in the past but I don’t want any online connectivity. But I’ll look into this. 
just want to use something that’s private and offline to control my workstation from another device the way audio gridder privately streams my plugins from one machine to the other

**Reply by apohl79** [reply]
VNC works on your local network.

**Reply by bsjdsp** [reply]
Is vnc free and or open source? 

I saw the website and I saw some pricing

**Reply by apohl79** [reply]
There is multiple free VNC servers and clients for different operating systems.
